src/LiveComponent/src/Form/Type/LiveCollectionType.php

```diff
@@ -92,6 +104,9 @@ public function configureOptions(OptionsResolver $resolver): void
             'button_add_options' => [],
             'button_delete_type' => ButtonType::class,
             'button_delete_options' => [],
+            'allow_add' => true,
+            'allow_delete' => true,
+            'by_reference' => false,
```

Comment:
Is it a good idea to change the defaults? I think it is more common this way.

Reply:
I think that's fine. If you don't want these, then there isn't a point to using `LiveCollectionType`.